Components/LandingPage/WhyChooseUs.tsx

```diff
@@ -31,16 +31,14 @@ const WhyChooseUs = () => {
   );
 };
 
-const InfoCard = ({ title, description }: any) => {
+const InfoCard = ({ title, description}: any) => {
   return (
-    <div className=" p-4 hover:bg-zinc-800 transition-colors font-archivo rounded-3xl ">
-      <div className="space-y-4 ">
+     <div className=" p-4 hover:bg-blue-400 transition-colors font-archivo rounded-3xl ">
```

Comment:
instead of blue, use text-zinc 100 for light theme and text-zinc-800 for dark theme

Reply:
I guess, you are saying about hover.
I have changed to  `hover:bg-zinc-200 dark:hover:bg-zinc-800`
I have used 200 cause 100 is so light and not even clearly visible.
Could you please review the changes now